lib/datadog/tracing/contrib/rails/patcher.rb

```diff
@@ -5,11 +5,22 @@
 require_relative 'middlewares'
 require_relative 'utils'
 require_relative '../semantic_logger/patcher'
+require_relative 'ext'
 
 module Datadog
   module Tracing
     module Contrib
       module Rails
+        # Patcher to trace rails routing done by JourneyRouter
+        module JourneyRouterPatch
+          def find_routes(*args)
+            result = super
+            integration_route = result.first[2].path.spec.to_s.gsub(/\(\.:format\)/, '') if result.any?
```

Comment:
If `result.any?` returns `false`, is it still required to push into `Thread.current[:datadog_http_routing]`?

Reply:
Depends on whether you want to record the trip to the router even though it failed to match, which can come in really useful when it comes to debugging.